.browserslistrc

```diff
@@ -1 +0,0 @@
-defaults
```

Comment:
This file was added when we webpacker-ified Argo, and we longer use webpacker for assets.